iroha/src/tx.rs

```diff
@@ -25,14 +28,17 @@ struct Payload {
     /// An ordered set of instructions.
     instructions: Vec<Instruction>,
     /// Time of creation (unix time, in milliseconds).
-    creation_time: String,
+    creation_time: u64,
+    /// The transaction will be dropped after this time if it is still in a `Queue`.
+    proposed_ttl_ms: u64,
```

Comment:
I think this field can be named `time_to_live_ms`

Reply:
TTL is quite a common term from network terminology, I guess it will be clear for developers/contributors?